src/palette.ml

```diff
@@ -43,6 +109,9 @@ let load_tic80_palette (raw : string) : t =
   else
     raise (Invalid_argument "Palette size must not be zero or negative")
 
+let load_lospec_palette (raw: string): t =
```

Comment:
I think this shouldn't be here - this is incomplete, so won't compile, and the feature would overlap with #27 which is being worked on by someone else.